stout/flags.cc

```diff
@@ -179,7 +179,7 @@ void Parser::Parse(
           "Failed to parse non-boolean flag '"
           + non_negated_name + "' via '" + name + "'");
       continue;
-    } else if (!value) {
+    } else if (!value.value().size()) {
```

Comment:
This looks like a bug, what if `value.has_value()` is `false`? Shouldn't this crash in that case? I'm guessing you ran into a test case that you wrote that didn't work correctly though hence you had to change this. Perhaps we need to do something like:

```suggestion
    } else if (!value.has_value() || value.value().empty()) {
```